apps/evm/lib/evm/configuration/eip158.ex

```diff
@@ -0,0 +1,49 @@
+defmodule EVM.Configuration.EIP158 do
+  defstruct fallback_config: EVM.Configuration.EIP150.new(),
+            exp_byte_cost: 50
+
+  def new do
+    %__MODULE__{}
+  end
+end
+
+defimpl EVM.Configuration, for: EVM.Configuration.EIP158 do
+  alias EVM.Configuration
+
+  @spec contract_creation_cost(Configuration.t()) :: integer()
+  def contract_creation_cost(config),
```

Comment:
I think we might be able to use `defdelegate` for a lot of these functions that are simply calling the `fallback_config`. Don't think it changes things much in practice, but just a thought if you feel like it'd be nicer.

Reply:
@germsvel  Can you give a simple example? It looks like `defdelegate` is used to delegate to a different module

Reply:
Hmmm, yeah seems you're right. I thought there was another way to use `defdelegate` but doesn't seem like it makes sense here.